lib/uk/ac/ebi/interpro/InterProScan.groovy

```diff
@@ -474,35 +475,88 @@ class InterProScan {
     }
 
     static void printHelp(appsConfig) {
+        def optionsMap = this.PARAMS
+            .findAll { it.description }
+            .collectEntries { opt ->
+                [
+                    (opt.name): [
+                        description: opt.description,
+                        metavar    : opt.metavar
+                    ]
+                ]
+            }
+
         def result = new StringBuilder()
-        result << "Usage: nextflow run ebi-pf-team/interproscan6 -profile <PROFILE> --input <FASTA> --datadir <DATADIR> \n\n"
+        result << "Usage: nextflow run ebi-pf-team/interproscan6 -profile <PROFILE> --input <FASTA> [--datadir <DATADIR>] [options]\n\n"
         result << "Mandatory parameters:\n"
-        result << "  -profile <PROFILE>: use this parameter to choose a configuration profile.\n"
+        result << this.formatOption("profile",
+                                    [
+                                        metavar    : "<PROFILE>", 
+                                        description: "Execution profile defining the runtime environment (e.g. slurm, lsf, docker)."
+                                    ],
+                                    " -")
```

Comment:
It's not mandatory when running locally on bare metal. Maybe this should be noted in the description?